airflow/utils/db.py

```diff
@@ -23,16 +23,18 @@
 from airflow import settings
 from airflow.configuration import conf
 # noinspection PyUnresolvedReferences
-from airflow.jobs.base_job import BaseJob  # noqa: F401  # pylint: disable=unused-import
-# noinspection PyUnresolvedReferences
 from airflow.models import (  # noqa: F401 # pylint: disable=unused-import
     DAG, XCOM_RETURN_KEY, BaseOperator, BaseOperatorLink, Connection, DagBag, DagModel, DagPickle, DagRun,
     DagTag, Log, Pool, SkipMixin, SlaMiss, TaskFail, TaskInstance, TaskReschedule, Variable, XCom,
 )
 # We need to add this model manually to get reset working well
 # noinspection PyUnresolvedReferences
 from airflow.models.serialized_dag import SerializedDagModel  # noqa: F401  # pylint: disable=unused-import
-from airflow.utils.session import create_session, provide_session  # noqa  # pylint: disable=unused-import
```

Comment:
Removing create_session from here is a User-facing API change -- it's not unheard of for custom operators to access Airflow's DB via this.

Reply:
Oh, I forgot that. I will revert it and add comment for future